wire-ios-sync-engine/Tests/Source/UserSession/ZMClientRegistrationStatusTests.swift

```diff
@@ -127,7 +120,29 @@ extension ZMClientRegistrationStatusTests {
         }
     }
 
-    func testThatItReturnsRegisteringMLSClient_IfIfMLSIsEnabledAfterRegisteringProteusClient() {
+//    func testThatItReturnsWaitingForE2EIStatus_IfMLSIsEnabledAfterRegisteringProteusClient() {
+//        syncMOC.performAndWait {
+//            // given
+//            let selfUser = ZMUser.selfUser(in: syncMOC)
+//            selfUser.remoteIdentifier = UUID()
+//            selfUser.emailAddress = "email@domain.com"
+//            let selfUserClient = createSelfClient()
+//            selfUserClient.remoteIdentifier = "clientID"
+//
+//            DeveloperFlag.storage = .temporary()
+//            DeveloperFlag.enableMLSSupport.enable(true)
+//            BackendInfo.storage = .temporary()
+//            BackendInfo.apiVersion = .v5
+//
+//            // when
+//            sut.didRegisterProteusClient(selfUserClient)
+//
+//            // then
+//            XCTAssertEqual(self.sut.currentPhase, .waitingForE2EIStatus)
+//        }
+//    }
```

Comment:
is this meant to be commented?

Reply:
It should be delete since the `.waitingForE2EIStatus` case no longer exists.